jax/numpy/lax_numpy.py

```diff
@@ -2726,14 +2738,18 @@ def polymul(a1, a2, *, trim_leading_zeros=False):
   if isinstance(a2, np.poly1d):
     a2 = asarray(a2)
   if trim_leading_zeros and (len(a1) > 1 or len(a2) > 1):
-    a1, a2 = _trim_zeros(a1), _trim_zeros(a2)
+    a1, a2 = _trim_zeros(a1, trim='f'), _trim_zeros(a2, trim='f')
   if len(a1) == 0:
     a1 = asarray([0.])
   if len(a2) == 0:
     a2 = asarray([0.])
   val = convolve(a1, a2, mode='full')
   return val
 
+@_wraps(np.trim_zeros)
+def trim_zeros(filt, trim='fb'):
+  return jit(_trim_zeros, static_argnums=(0,1))(filt, trim)
```

Comment:
When all arguments are static, there's nothing gained by jit-compiling the function. It would make more sense to directly define the implementation here.